scss/_buttons.scss

```diff
@@ -50,6 +50,7 @@
   [class*="svg-"],
   [class*="icon-"] {
     display: inline-block;
+    transform: translateY(1px);
```

Comment:
If you make this modification here, you will also move down the icon when the button is only composed of an icon. I suppose you rather need to modify the content of the following selector:

```
&:not(.btn-icon) {
    [class*="svg-"],
    [class*="icon-"] {
```

On Firefox, when I enable/disable `transform: translateY(1px);`, I can see the icon slightly also moving on the x-axis. I don't have this behavior with:

```scss
&:not(.btn-icon) {
  [class*="svg-"],
  [class*="icon-"] {
    margin-right: $border-width * 3;
    position: relative;
    top: 1px;
  }
}
```

However, I haven't tried it in all browsers